.goreleaser.yaml

```diff
@@ -82,6 +82,34 @@ builds:
     asmflags: *build-asmflags
     gcflags: *build-gcflags
     ldflags: *build-ldflags
+  - id: darwin-amd64
```

Comment:
We can't build macos binaries via goreleaser because the macos compiler is not available (and not licensed for use) on Linux.

So the macos binary builds need to continue to be run on `macos-latest` machines via the github action. And then rather than using `goreleaser`'s github release integration, we'll have to tie all this together via github actions to create the release and upload the assets, similar to what we're doing in ` .github/workflows/release.yaml`